core/src/main/java/org/apache/brooklyn/core/mgmt/persist/BrooklynMementoPersisterToObjectStore.java

```diff
@@ -674,6 +685,22 @@ private void delete(String subPath, String id, PersistenceExceptionHandler excep
         }
     }
 
+    private void updatePlaneId(String planeId, PersistenceExceptionHandler exceptionHandler) {
+        try {
+            if (planeId==null) {
+                LOG.warn("Null content for planeId");
+            }
+
+            String persistedPlaneId = read(PLANE_ID_FILE_NAME);
+            if (persistedPlaneId != null && !persistedPlaneId.equals(planeId)) {
+                throw new IllegalStateException("Persisted planeId found (" + persistedPlaneId + ") but instance planeId is different (" + planeId + "). Overwriting!");
```

Comment:
"Overwriting!"?  does throwing the exception here not in fact skip the overwrite?  As per your comment on the description of the PR, "don't overwrite persisted store planeId if different"?